tests/pulp_test.py

```diff
@@ -0,0 +1,23 @@
+import testinfra
+
+def test_pulp_service(host):
+    pulp = host.service("pulp")
+    assert pulp.is_running
+    assert pulp.is_enabled
```

Comment:
A good rule when writing tests is that you want a single assertion per test. That way you get all the needed output reported at once so I'd recommend:
```suggestion
    assert pulp.is_running and pulp.is_enabled
```

But I'd go a bit further and try:
```suggestion
    assert pulp.exists and pulp.is_running and pulp.is_enabled
```

And I'd also play with it and see how it reports a (partial) failure. For example, force it to fail with 
```suggestion
    assert not pulp.exists and pulp.is_running and pulp.is_enabled
```

I think pytest will report all variables, but that's useful to know.